ui_feature.go

```diff
@@ -126,10 +176,25 @@ func (f *UIFeature) thePageShouldHaveTheFollowingContent(expectedAPIResponse *go
 	return f.StepError()
 }
 
-func (f *UIFeature) runWithTimeOut(ctx *context.Context, timeout time.Duration, tasks chromedp.Tasks) chromedp.ActionFunc {
+func (f *UIFeature) RunWithTimeOut(ctx *context.Context, timeout time.Duration, tasks chromedp.Tasks) chromedp.ActionFunc {
```

Comment:
The ctx parameter is not being used
```suggestion
func (f *UIFeature) RunWithTimeOut(timeout time.Duration, tasks chromedp.Tasks) chromedp.ActionFunc {
```